controllers/wizard/fileUpload/form/PKPSubmissionFilesUploadBaseForm.php

```diff
@@ -262,8 +263,15 @@ public function getRevisionSubmissionFilesSelection($user, $uploadedFile = null)
                 ->withUserId($user->getId())
                 ->exists();
 
+            $allowedFileStages = [
+                SubmissionFile::SUBMISSION_FILE_REVIEW_ATTACHMENT,
+                SubmissionFile::SUBMISSION_FILE_REVIEW_FILE,
+                SubmissionFile::SUBMISSION_FILE_INTERNAL_REVIEW_FILE,
+            ];
+
+            // #11341 Might not be necessary, as the fileType is now filtered also for review stage in getSubmissionFiles
             if (
-                ($submissionFile->getFileStage() == SubmissionFile::SUBMISSION_FILE_REVIEW_ATTACHMENT || $submissionFile->getFileStage() == SubmissionFile::SUBMISSION_FILE_REVIEW_FILE) &&
+                in_array($submissionFile->getFileStage(), $allowedFileStages) &&
```

Comment:
Adding `SUBMISSION_FILE_INTERNAL_REVIEW_FILE ` from OMP.  But as mentioned in the comment, might not be necessary